Common/Python/PandasData.cs

```diff
@@ -60,199 +59,242 @@ public PandasData(object data)
             {
                 using (Py.GIL())
                 {
-                    _pandas = Py.Import("pandas");
-
                     // this python Remapper class will work as a proxy and adjust the
                     // input to its methods using the provided 'mapper' callable object
-                    _remapperFactory = PythonEngine.ModuleFromString("remapper",
-                        @"import wrapt
-import pandas
+                    _pandas = PythonEngine.ModuleFromString("remapper",
+                        @"import pandas as pd
+from pandas.core.resample import Resampler, DatetimeIndexResampler, PeriodIndexResampler, TimedeltaIndexResampler
+from pandas.core.groupby.generic import DataFrameGroupBy, SeriesGroupBy
+from pandas.core.indexes.frozen import FrozenList as pdFrozenList
+from pandas.core.window import Expanding, EWM, Rolling, Window
+from inspect import getmembers, isfunction, isgenerator
+from functools import partial
+from sys import modules
+
 from clr import AddReference
 AddReference(""QuantConnect.Common"")
-from QuantConnect import *
-
-originalConcat = pandas.concat
-
-def PandasConcatWrapper(objs, axis=0, join='outer', join_axes=None, ignore_index=False, keys=None, levels=None, names=None, verify_integrity=False, sort=None, copy=True):
-    return Remapper(originalConcat(objs, axis, join, join_axes, ignore_index, keys, levels, names, verify_integrity, sort, copy))
+from QuantConnect import Symbol, SymbolCache
```

Comment:
I would suggest avoiding partial C# imports to avoid the issue https://github.com/QuantConnect/Lean/issues/4249